test/server/chat-plugins/hosts.js

```diff
@@ -6,13 +6,9 @@
 'use strict';
 
 const assert = require('assert').strict;
-let hosts;
+const hosts = require('../../../.server-dist/chat-plugins/hosts');
```

Comment:
I'm fairly sure this change can also be done without polluting tests global scope with ``Config``.

Reply:
Yes, a try/except makes that simple:
```ts
process.nextTick(() => {
	// Wrap this in a try/except because Chat isn't defined in unit tests
	try {
		Chat.multiLinePattern.register('/(datacenters|datacenter|dc|iprange|ipranges) (add|widen|remove)');
	} catch (e) {
		if (!(e instanceof ReferenceError)) throw e;
	}
});
```